app/helpers/application_helper.rb

```diff
@@ -563,11 +563,16 @@ def creator_ids_for_css_classes(creation)
   def css_classes_for_creation_blurb(creation)
     return if creation.nil?
 
-    Rails.cache.fetch("#{creation.cache_key_with_version}/blurb_css_classes-v2") do
-      creation_id = creation_id_for_css_classes(creation)
-      creator_ids = creator_ids_for_css_classes(creation).join(" ")
-      "blurb group #{creation_id} #{creator_ids}".strip
-    end
+    creation_id = creation_id_for_css_classes(creation)
+    creator_ids = if creation.try(:pseuds)&.loaded?
+                    creator_ids_for_css_classes(creation).join(" ")
+                  else
+                    Rails.cache.fetch("#{creation.cache_key_with_version}/blurb_css_classes-v2") do
```

Comment:
The cache format changed because you pulled the last line out of this cache call, so the cache key needs to change. Though now I'm wondering whether this will affect how accurately we can measure the performance impact

Reply:
I think observing the longer term performance should be okay.